pl_examples/hydra_examples/conf/optimizer.py

```diff
@@ -0,0 +1,108 @@
+from dataclasses import dataclass
+
+from hydra.core.config_store import ConfigStore
+from hydra.types import ObjectConf
+from typing import Optional
+
+cs = ConfigStore.instance()
+
+
+@dataclass
+class AdamConf:
+    betas: tuple = (0.9, 0.999)
+    lr: float = 1e-3
+    eps: float = 1e-8
+    weight_decay: float = 0
+    amsgrad: bool = False
+
+
+cs.store(
+    group="opt", name="adam", node=ObjectConf(cls="torch.optim.Adam", params=AdamConf()),
+)
+cs.store(
+    group="opt", name="adamw", node=ObjectConf(cls="torch.optim.AdamW", params=AdamConf()),
+)
```

Comment:
nice.
does not matter much, but you can also reuse the same object in both calls.